cmake/modules/FindUBSan.cmake

```
@@ -0,0 +1,13 @@
+set(UBSan_LIB_NAME UBSan)
+
+find_library(UBSan_LIBRARY
+  NAMES libubsan.so libubsan.so.5 libubsan.so.4 libubsan.so.3 libubsan.so.2 libubsan.so.1 libubsan.so.0
```

Seems there are no other versions yet.
```suggestion
  NAMES libubsan.so libubsan.so.1 libubsan.so.0 libubsan.so.0.0.0
```